core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -257,15 +254,21 @@ class KafkaServer(val config: KafkaConfig, time: Time = SystemTime, threadNamePr
 
     if (chroot.length > 1) {
       val zkConnForChrootCreation = config.zkConnect.substring(0, config.zkConnect.indexOf("/"))
-      val zkClientForChrootCreation = ZkUtils.createZkClient(zkConnForChrootCreation, config.zkSessionTimeoutMs, config.zkConnectionTimeoutMs)
-      ZkUtils.makeSurePersistentPathExists(zkClientForChrootCreation, chroot)
+      val zkClientForChrootCreation = ZkUtils.create(zkConnForChrootCreation, 
+                                                     config.zkSessionTimeoutMs,
+                                                     config.zkConnectionTimeoutMs,
+                                                     ToolsUtils.isSecure(System.getProperty("java.security.auth.login.config")))
```

Comment:
I think we should not repeat this string everywhere. We should either introduce a constant for "java.security.auth.login.config" or a method that returns the system property (for example `authLoginConfigProperty`).